### ISSUE TITLE
TravelAgent not available for PlayerPortalEvent in 1.14.4

### ISSUE THREAD

**Issue by TechTailor** [Multiverse/Multiverse-Core#2130]
Hello,

I know this is sort of a duplicate of issue #2037 but that issue was closed with the notion that it doesnt effect performance and the error/warning can be ignored, however, i am running a 1.14.4 Paper (build 225) and whenever someone uses a Nether Portal I get that error and the whole server lags up for a second (basically comes to a stall). 

```
[WARNING] .... [Multiverse-Core] TravelAgent not available for PlayerPortalEvent for SFBC
[ERROR] .... --- DO NOT REPORT THIS TO PAPER - THIS IS NOT A BUG OR A CRASH - git-Paper-225 (MC: 1.14.4) ---
[ERROR] .... The server has not responded for 10 seconds! Creating thread dump
[ERROR] .... ------------------------------
[ERROR] .... Server thread dump (Look for plugins here before reporting to Paper!):
[ERROR] .... The server is waiting on these chunks: []
[ERROR] .... ------------------------------
[ERROR] .... Current Thread: Server thread
[ERROR] .... PID: 31 | Suspended: false | Native: false | State: RUNNABLE
[ERROR] .... Stack:
[ERROR] .... net.minecraft.server.v1_14_R1.PortalTravelAgent.a(PortalTravelAgent.java:80)
[ERROR] .... net.minecraft.server.v1_14_R1.PortalTravelAgent.a(PortalTravelAgent.java:33)
[ERROR] .... net.minecraft.server.v1_14_R1.EntityPlayer.a(EntityPlayer.java:941)
[ERROR] .... net.minecraft.server.v1_14_R1.Entity.doPortalTick(Entity.java:2215)
[ERROR] .... net.minecraft.server.v1_14_R1.Entity.entityBaseTick(Entity.java:398)
[ERROR] .... net.minecraft.server.v1_14_R1.EntityLiving.entityBaseTick(EntityLiving.java:240)
[ERROR] .... net.minecraft.server.v1_14_R1.Entity.tick(Entity.java:370)
[ERROR] .... net.minecraft.server.v1_14_R1.EntityLiving.tick(EntityLiving.java:2285)
[ERROR] .... net.minecraft.server.v1_14_R1.EntityHuman.tick(EntityHuman.java:165)
[ERROR] .... net.minecraft.server.v1_14_R1.EntityPlayer.playerTick(EntityPlayer.java:426)
[ERROR] .... net.minecraft.server.v1_14_R1.PlayerConnection.tick(PlayerConnection.java:146)
[ERROR] .... net.minecraft.server.v1_14_R1.NetworkManager.a(NetworkManager.java:260)
[ERROR] .... net.minecraft.server.v1_14_R1.ServerConnection.c(ServerConnection.java:139)
[ERROR] .... net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1255)
[ERROR] .... net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:417)
[ERROR] .... net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1098)
[ERROR] .... net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:925)
[ERROR] .... java.lang.Thread.run(Thread.java:813)
[ERROR] .... ------------------------------
[ERROR] .... --- DO NOT REPORT THIS TO PAPER - THIS IS NOT A BUG OR A CRASH ---
[ERROR] .... ------------------------------
[INFO] .... SFBC has made the advancement [We Need to Go Deeper]
[WARNING] .... Can't keep up! Is the server overloaded? Running 9979ms or 199 ticks behind
```

Would appreciate if a workaround or solution for this issue could be provided.
